core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -444,12 +444,15 @@ private[spark] object JsonProtocol {
     ("Disk Size" -> rddInfo.diskSize)
   }
 
-  def storageLevelToJson(storageLevel: StorageLevel): JValue = {
-    ("Use Disk" -> storageLevel.useDisk) ~
-    ("Use Memory" -> storageLevel.useMemory) ~
-    ("Deserialized" -> storageLevel.deserialized) ~
-    ("Replication" -> storageLevel.replication)
-  }
+  def storageLevelToJson(storageLevel: StorageLevel): JValue =
```

Comment:
Braces.

Reply:
Sorry, missed it. I've decided not to add braces to `storageLevelFromJson` because it seems to look OK with the toplevel match.